catalyst/data/cv/__init__.py

```diff
@@ -23,6 +23,7 @@
         TensorToImage,
         ImageToTensor,
     )
+    from catalyst.data.cv.transforms.kornia import OneOf, OneOfV2
```

Comment:
what do you think about naming? I don't think V1/V2 are easy names to understand :)
`BatchOneOf`/`SampleOneOf`? or maybe just add ``mode`` param to the `OneOf`?